handler/interceptors.go

```diff
@@ -372,9 +375,47 @@ func GetSingleValue(md metadata.MD, key string) (value string, err *GrpcError) {
 // NoOpInterceptor is a gRPC interceptor which doesn't do payment checking.
 func NoOpInterceptor(srv interface{}, ss grpc.ServerStream, info *grpc.StreamServerInfo,
 	handler grpc.StreamHandler) error {
+	if config.GetBool(config.AllowedUserFlag){
```

Comment:
Why additional kind of payment metadata manually handled here? If we need to check metadata for such calls then it should be additional payment type because it is actually is. And if there is no case for `NoOpInterceptor` it should be deleted.